### PR TITLE
List recipes of user

### LINKED ISSUE #7: [1.5pt] Recipes list (User)
- Should display a list of recipes created by the logged-in user as in the wireframe.
- Should lead to recipe details.
- If the user is the owner of the recipe, should allow the user to delete it.

### PR DESCRIPTION
**Pull request by MattGomb**
Hi @jfoyarzo please check the changes, let me know if you'd like to see anything else in this milestone!

### CODE REVIEW

File: app/controllers/recipes_controller.rb [hunk 1]
```diff
@@ -19,15 +20,14 @@ def edit; end
 
   # POST /recipes or /recipes.json
   def create
-    @recipe = Recipe.new(recipe_params)
+    @recipe = Recipe.new(params.require(:recipe).permit(:name, :preparation_time, :cooking_time, :description, :public))
```
Comment (jfoyarzo) [inline app/controllers/recipes_controller.rb]: Hi @MattGomb ! great job so far! For this line, you could use the private method we have defined in line 64

**Review by jfoyarzo — CHANGES_REQUESTED** [review]
Hey @MattGomb , great job 👍🏼 ! I only had one small observation, the code look great 😃

File: app/controllers/recipes_controller.rb (hunk 1 above)
Comment (MattGomb) [reply app/controllers/recipes_controller.rb]: done!

**Review by jfoyarzo — APPROVED** [review]
Great job! ready to merge 😺

**Pull request merged**